### DISCUSSION TITLE
KEDA tcp lookup no such host can't reach service for ScaledObject

### DISCUSSION THREAD

**Discussion by kbfirebreather** [kedacore/keda#3129] [Q&A / Need Help] [answered]
Trying to use KEDA on a K8S rancher configured AWS cluster that has a dozen or so namespaces. Added a KEDA namespace, and installed KEDA. Deployed my application to my namespace with a keda.sh/v1alpha1 ScaledObject. The logs in the operator are showing the following:

```
1.6545237767994993e+09  ERROR   scalehandler    Error getting scale decision    {"scaledobject.Name": "artemis-scaledobject", "scaledObject.Namespace": "<namespace>", "scaleTarget.Name": "<release>-compute-node", "error": "Get \"http://<release>-artemis:8161/console/jolokia/read/org.apache.activemq.artemis:broker=%220.0.0.0%22,component=addresses,address=%22<address>%22,subcomponent=queues,routing-type=%22anycast%22,queue=%22<queue>%22/MessageCount\": dial tcp: lookup <release>-artemis on 10.43.0.10:53: no such host"}
```

I obfuscated some of the information in the log, the log output does not actually say `<release>` or `<queue>`, etc. When running KEDA / my application with the ScaledObject in K3S, I have no issues. 

Is there something I need to do to the KEDA namespace to get the operator to be able to work as intended? Thank you for any help!

**Comment by tomkerkhove** [comment]
This is most probably a networking/security policy in place that is blocking the communication. Did you verify this?

**Comment by kbfirebreather** [comment, marked as answer]
It was PBCAK. I didn't realize for the managementEndpoint I needed to include the namespace as part of it. I had `<release>-artemis:8161` when I needed `<release>.<namespace>-artemis:8161`

**Reply by zroubalik** [reply]
yeah, that's always important to do! Glad it is working for you :)
